shared/spotify-api/src/types.ts

```diff
@@ -97,7 +97,7 @@ export namespace Spotify {
       href: string;
       id: string;
       name: string;
-      type: 'artist';
+      type: 'artist' | 'show';
```

Comment:
I don't think this is quite right. There shouldn't be a case where you get an "artist" that happens to be a show.

If I were to guess, I believe this would come from the fact that a playlist can contain both episodes and tracks, so the type is a little bit weird/special for this use case. Could we instead create a specific type for that endpoint that matches this type? I could have sworn I had this captured correctly when I went through all the endpoints the first time, so this change surprised me.

Reply:
Huh, deleting this didn't cause any problems, it must have been me trying to treat a symptom early that I fixed for real later 😓